src/validations/loginValidations.js

```diff
@@ -1,9 +1,17 @@
+import strings from 'localization';
+
+const { VALIDATIONS } = strings;
+
 export default {
   email: {
     presence: true,
-    email: true,
+    email: { message: VALIDATIONS.email.email },
```

Comment:
I got the idea here but It's not that clear maybe you can rename it to
```suggestion
    email: { message: VALIDATIONS.email.format },
```